content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -34,20 +34,20 @@ proxy is different from configuring traffic to external HTTP and HTTPS services.
     $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
     {{< /text >}}
 
-    Note that any pod that you can `exec` and `curl` from would do.
+    Note that you can use as a test source any pod that has `curl` installed.
 
-*   Create a shell variable to hold the name of the source pod for sending requests to external services.
-    If you used the [sleep]({{<github_tree>}}/samples/sleep) sample, run:
+*   Create the `SOURCE_POD` environment variable to store the name of the source pod for sending requests to external
+    services:
 
     {{< text bash >}}
     $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
     {{< /text >}}
 
 ## Deploy an HTTPS proxy
 
-For this example, to simulate a legacy proxy, you deploy an HTTPS proxy inside your cluster. Also, to simulate a more
-realistic proxy that is running outside of your cluster, you will address the pod of the proxy by its IP address and
-not by a Kubernetes service.
+For the sake of this example only, to simulate a legacy proxy, you deploy an HTTPS proxy inside your cluster.
+Also, to simulate a more realistic proxy that is running outside of your cluster, you will address the pod of the proxy
```

Comment:
Avoid using words with little to no semantic value to simplify your text. Use the imperative present tense instead of the future tense. Lastly, I suggest using the possessive to move the noun "pod" closer to the pronoun "its" to make the reference more evident:
 ```suggestion
To simulate a more realistic proxy running outside of your cluster, address the proxy's pod
```

Reply:
`Also` is added to connect with the previous sentence:

> To simulate a legacy proxy... Moreover, [to be closer to the real life situation], you will address the proxy's pod by its IP address

Please advise.

Reply:
> Use the imperative present tense instead of the future tense

The imperative could confuse the readers - they can ask themselves: "What should I do to address the proxy's pod by its IP address?". 

The meaning here is "in the following instructions, when you will address the pod, you will address it by its IP address, as we will instruct you".

Please advise.

Reply:
Alternatively, the meaning is: in the following steps you will address the pod by its IP, do not be surprised - you will do it to simulate a real life situation better.

Reply:
Let me address your concerns in order:
1. The use of `Also`, `Moreover`, and similar sentence connecting words:
As we learn to write, we are taught that the sentences in a paragraph must be linked using these words. What most of us forget, is that we are also taught to do so sparingly and that not all lead words are the same. Readers connect sentences in a paragraph with an implicit `AND`. My advice to you would be to only use leading words when the connection differs from `AND`. For example, to connect two sentences in a paragraph with `BUT` I would use "However, ..."
2. The use of imperative:
I agree fully, readers will ask  "How do I address the proxy's pod by its IP address?". If you know this, my advise would be to link to the instructions on how to do that. The imperative will help you identify those questions and then ask: Will my audience know the answer or not? If not, link to the answer, define it briefly or link to a definition. 
3. The future meaning conflicts with the use of present tense: 
I understand that from your perspective, you are describing the actions readers will take and, thus, feel the need to use the future tense in your text. However, I would advise you to switch from a descriptive approach, writing a descriptions of the future actions to take, to a prescriptive approach, writing the actions readers need to take at that moment. What action do readers need to take at the beginning of your document: Keep reading! Therefore, I suggest:
Keep reading to learn how to address the pod by its IP and how this approach allows you to simulate a real life situation better.

Hopefully, I've been clear enough and answered your questions. Let me know if further discussion is needed.

Reply:
Note that addressing the pod by IP is done as part of the following steps. The sentence means: when you will perform the steps below, note that you will address the pod by IP (as part of the steps, not for the sake of addressing the pod by IP).